### ISSUE TITLE
Question: Autoload Archive

### ISSUE THREAD

**Issue by peter1000** [webrecorder/wabac.js#8]
hi,
thanks a lot for this code. It is the only thing I found which works fast enough on Windows 10 to be actual useful.

*Background:* I have a number of bought application which only have online documentation.
I have to run them on an offline Windows machine.

**My idea is**: to fetch a snapshot of the web-documentation in `.warc.gz` format. and than use this as my doumentation live on my offline PC.

**QUESTION:** do you think it is possible to automatically load a `.warc.gz`  file lets say: I execute a *bat file* which starts the server and loads the `.warc.gz` automatically?

If so could you give me some pointers on how to do this.

*Basically I would like to have*: one folder per documentation.warc.gz with a bat / or linux .sh file which executes all so that I can read the documentation with the defined starting url

Thanks for any idea

**Comment by peter1000** [comment]
sorry I had overlooked the info about: Dynamic Collections API

**Comment by ikreymer** [comment]
Yes, that should be possible with the `coll_<name>=<pathtowarc>` and `url=` params.

I am curious about your use case.. This is to launch offline documentation on demand that you have stored in WARC files?

It wouldn't be too hard to build an 'index' that could also prelaunch these as well.. This is still a prototype, so just curious about possible applications!
